src/controller/java/AndroidOperationalCredentialsIssuer.cpp

```diff
@@ -63,15 +63,15 @@ CHIP_ERROR AndroidOperationalCredentialsIssuer::Initialize(PersistentStorageDele
     ReturnErrorOnFailure(ASN1ToChipEpochTime(effectiveTime, mNow));
 
     Crypto::P256SerializedKeypair serializedKey;
-    uint16_t keySize = static_cast<uint16_t>(sizeof(serializedKey));
+    uint16_t keySize = static_cast<uint16_t>(serializedKey.Capacity());
 
     if (storage.SyncGetKeyValue(kOperationalCredentialsIssuerKeypairStorage, &serializedKey, keySize) != CHIP_NO_ERROR)
```

Comment:
keySize seems to be an In/Out parameter. Should we do `serializedKey.SetLength` call to set the length after reading?